packages/vscode-honey-css-modules/package.json

```diff
@@ -28,21 +28,31 @@
   "categories": [
     "Other"
   ],
-  "activationEvents": [],
+  "activationEvents": [
+    "onLanguage:css"
+  ],
```

Comment:
This is necessary to execute the activate function in `packages/vscode-honey-css-modules/src/index.cts` when the user open *.css in VS Code.